Cargo.toml

```diff
@@ -51,8 +51,8 @@ stm32f042 = ["stm32f0/stm32f0x2"]
 stm32f030 = ["stm32f0/stm32f0x0"]
 stm32f030x4 = ["stm32f030x6"]
 stm32f030x6 = ["stm32f030"]
-stm32f030x8 = ["stm32f030", "stm32f0/stm32f0x0"]
-stm32f030xc = ["stm32f030", "stm32f0/stm32f0x0"]
+stm32f030x8 = ["stm32f030"]
+stm32f030xc = ["stm32f030"]
```

Comment:
Snuck in another unrelated change, eh? ;)

Reply:
It's technically not *completely* unrelated